LICENCE.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+<!-- See http://creativecommons.org/licenses/by-sa/4.0/legalcode -->
+<!-- Human-readable summary: http://creativecommons.org/licenses/by-sa/4.0/ --> 
+<head>
+  <title>Creative Commons — Attribution-ShareAlike 4.0 International — CC BY-SA 4.0</title>
```

Comment:
OK, I've adapted the readme.md. I will also remove the local licence file. (I don't know if this will also remove these comments.)